### ISSUE TITLE
Firmware versions

### ISSUE THREAD

**Issue by tejeez** [fventuri/libsddc#4]
The firmware binary in the repository seems to be incompatible with the library, or at least I couldn't get sddc_stream_test working properly (tried on two different computers).

I finally got it working using firmware from https://groups.io/g/NextGenSDRs/topic/binary_builds_of_extio_sddc/78738204?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,78738204
and the latest commit 967eb2f4acb507c5f60a2ba6eabdf46db61ad7f4.

I think I tried all 4 combinations of that groups.io firmware, older firmware from the repository, commit 967eb2f4acb507c5f60a2ba6eabdf46db61ad7f4 and commit 5139e6ace947306301b80651a554c2a3f2cc91fb, and the combination above was the only one that worked for me.

Maybe it's something to do with "rejected 1 configuration due to insufficient available bus power" errors I got in dmesg with the old firmware, although I could apparently circumvent the problem with a trick from: https://blog.famzah.net/2010/08/11/usb-rejected-1-configuration-due-to-insufficient-available-bus-power/

But anyway, the latest firmware from that groups.io link didn't give those errors either, so the problem seems to be fixed already.

To avoid confusion like this, maybe the firmware binary in the repository should be updated, or just deleted and replaced with a link to a working version.

I just got the FX3 Explorer Kit a few days ago and now I'm just trying to get started, so it's possible that I'm doing something wrong too!

**Comment by fventuri** [comment]
Good catch @tejeez - last night when I updated the source code to use the latest 'SDDC_FX3.img' firmware image, I completely forgot that my repo contains a copy of that firmware image, and this is what caused all your problems - sorry for the confusion!

I just pushed to this repo the current firmware image from here https://github.com/ik1xpv/ExtIO_sddc/blob/master/ExtIO_sddc/SDDC_FX3.img, and hopefully this take care of your immediate issue, but I like your idea of replacing the copy of the firmware with a link to the latest version instead.

I don't have an RX888 here to try things out, so my most recent code changes might contain a bug or two (they should compile without any errors or warnings though).
The good news here is that these days I am working with @howard0su to move this low level library 'libsddc' together with all the existing code in the 'ExtIO_sddc' repo (https://github.com/ik1xpv/ExtIO_sddc), so that all the code will be consistent and well tested (and hopefully avoid problems like you had today).

Again, sorry for the problems with the old firmware image.
Franco
